src/model/track-bounds.ts

```diff
@@ -0,0 +1,47 @@
+import { TrackPath } from './ChallengeConfiguration'
+
+export const trackBounds = (
```

Comment:
**Suggested Change**
I remember we wrote these reducers together, but I think you met with Tyler and greatly simplified this using `Math.max()`? 📈

I'm going to mark this as a suggested change, because I think that is a lot better and would like to see tit.

Reply:
We will implement that later, I would prefer the working version in there for now. I will be saving the refactored version.

Reply:
![Screenshot from 2021-08-06 17-34-29](https://user-images.githubusercontent.com/19766927/128573378-6a6dfdb0-7a87-40b6-93ac-a0ac5ae3fe52.png)

Talked in slack; marking as resolved 👍